cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -388,7 +389,7 @@ func (sub EthereumSub) logToEvent(clientChainID *big.Int, contractAddress common
 		return event, false, nil
 	}
 
-	err := contractABI.Unpack(&event, eventName, cLog.Data)
```

Comment:
@iverc look like the event is returned here instead of decoded into the variable above, so it probably needs to be assigned to `event, err := contractABI.Unpack()`